src/openms/include/OpenMS/QC/Ms2IdentificationRate.h

```diff
@@ -102,6 +102,8 @@ namespace OpenMS
      * @return Status for RAWMZML and POSTFDRFEAT
      */
     QCBase::Status requires() const override;
+
+    void addMetaDataMetricsToMzTab(MzTabMetaData& meta);
```

Comment:
hmm don't know if I would add the dependency to MzTab to this class.
if you don't want to go with a new class that takes QC results and annotates in mzTab then 
I would suggest making this method static.

Reply:
But it is using the results from inside this class instance. I don't think it can be static, can it?

Reply:
I guess I could forward declare the class to save the import

Reply:
What about putting it into MzTab? Then it's just the other way around I guess.

Reply:
Not too important right now. Alternatively one could also think about a more low level return type instead of annotating directly. returning a String or StringList or key value pairs maybe. Don’t know if that is too cumbersome though